hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/token/TokenCreateSpecs.java

```diff
@@ -771,7 +788,7 @@ public HapiSpec onlyValidCustomFeeScheduleCanBeCreated() {
                 .then(getTokenInfo(token)
                         .hasCustom(fixedHbarFeeInSchedule(hbarAmount, hbarCollector))
                         .hasCustom(fixedHtsFeeInSchedule(htsAmount, feeDenom, htsCollector))
-                        .hasCustom(fixedHtsFeeInSchedule(htsAmount, token, htsCollector))
+                        .hasCustom(fixedHtsFeeInSchedule(htsAmount, SENTINEL_VALUE, htsCollector))
```

Comment:
Seems that the above tokeCreate was updated but the getTokenInfo was not.

Reply:
I am not sure I understand. Why was this change necessary?

Reply:
+1

Reply:
I had the wrong impression of what the test did. I will give it another go and try to fix it 👍

Reply:
Okay, I reverted the change and created an issue for that since I can't find what's wrong with this test. https://github.com/hashgraph/hedera-services/issues/8811